app/2.0/docs/tools/polymer-json.md

```diff
@@ -138,29 +138,66 @@ the `builds` property. This is equivalent to passing different CLI flags to the
-* `swPrecacheConfig`: An optional configuration file for the generated service worker.
-* `insertPrefetchLinks`: If `true`, insert prefetch link elements into your fragments so that all
-dependencies are prefetched immediately.
-* `bundle`: May be boolean, or an object bag of settings matching specified [`polymer-bundler`](https://github.com/Polymer/polymer-bundler) constructor options. This functionality emulates the behavior of the Polymer 1.x build tools.
-  * If `true`, bundle your application.
-  * If an object is provided with further configuration, the options are:
-    * `excludes`: Optional, `String[]`. Excludes the listed paths of files or folders from inlining.
-    * `inlineCss`: Optional, `Boolean`. Specifies whether to inline external CSS file contents into `<style>` tags.
-    * `inlineScripts`: Optional, `Boolean`. Specifies whether to inline external JavaScript file contents into `<script>` tags.
-    * `rewriteUrlsInTemplates`: Optional, `Boolean`. Specifies whether to rewrite element attributes inside of templates when inlining HTML.
-    * `sourcemaps`: Optional, `Boolean`. Specifies whether to create identity source maps for inline scripts.
-* `html`:
-  * `minify`: If `true`, minify all HTML.
-* `css`:
-  * `minify`: If `true`, minify all CSS.
-* `js`:
-  * `minify`: If `true`, minify all JS.
-  * `compile`: If `true`, use babel to compile all ES6 JS down to ES5.
+*   `name`: An optional name for your build. If multiple builds are defined, the `name` property 
+    is required.
+
+*   `preset`: An optional preset name that your build configuration can inherit from. 
+    See below for more information.
+
+*   `addServiceWorker`: If `true`, generate a service worker for your application.
+
+*   `addPushManifest`: If `true`, generate an [HTTP/2 Push Manifest](https://github.com/GoogleChrome/http2-push-manifest) for your application.
+
+*   `swPrecacheConfig`: An optional configuration file for the generated service worker.
+
+*   `insertPrefetchLinks`: If `true`, insert prefetch link elements into your fragments so that 
+    all dependencies are prefetched immediately.
+
+*   `bundle`: You may provide a boolean (`true` to bundle your application; default is `false`), 
+    or an object containing bundling configuration options.
+    
+    *   `excludes`: A list of paths of files and/or folders that should not be inlined. 
+      
+    *   `inlineCss`: Inline external CSS file contents into `<style>` tags.
+
+    *   `inlineScripts`: Inline external Javascript file contents into `<script>` tags.
+    
+    *   `rewriteUrlsInTemplates`: Rewrite element attributes inside templates when inlining html.
+
+    *   `sourcemaps`: Create identity source maps for inline scripts.
+
+    *   `stripComments`: Remove all comments except those tagged `@license`, or starting with
+        `<!--!` or `<!--#`.
+
+*   `html`:
```

Comment:
should these say they're an object to be extra clear about what a nested item means?